.pre-commit-config.yaml

```
@@ -0,0 +1,44 @@
+exclude: (?x)(\.txt)
+repos:
+-   repo: https://github.com/psf/black
```

```suggestion
-   repo: https://github.com/myint/autoflake
    rev: v1.5.3
    hooks:
    -   id: autoflake
        name: autoflake
        args: ["--in-place", "--remove-unused-variables", "--remove-all-unused-imports"]
        language: python
        files: \.py$
-   repo: https://github.com/psf/black
```

Auto flake is great for removing unused imports and variables. 😁 

(Note the diff adds `autoflake` before black, might be easy to miss)

---

cool! gonna add it